utils/crates-io/src/publisher.rs

```diff
@@ -114,6 +114,11 @@ impl Publisher {
             panic!("Packages {failed:?} failed to pass the check ...");
         }
 
+        // Test if gtest works
+        if !crate::test("demo-syscall-error")?.success() {
```

Comment:
it would be nice to test `gtest` and `gclient` together

Reply:
we currently don't have examples using `gclient` for testing and since `gclient` has no substrate dependencies, it is safe from the dependency issue, it works when it can be compiled

Reply:
we have substrate dependencies in dependency tree: `gclient -> gsdk -> sp-core`. what if we change `gsdk` and it uses some unpatched dependency?

Reply:
makes sense, actually I can embed a test of `gclient` in `crates-io check` to verify it as well